napari/layers/_tests/test_layer_actions.py

```diff
@@ -86,3 +86,28 @@ def test_convert_dtype(mode):
 
     assert ll[-1].data[5, 5] == 1000
     assert ll[-1].data.flatten().sum() == 1000
+
+
+@pytest.mark.parametrize('order', [(0, 1, 2, 3), (3, 1, 2, 0), (2, 1, 0, 3)])
+def test_split_stack(make_napari_viewer, order) -> None:
+    axis = 0
+    larger_data = np.zeros((2, 3, 4, 5), dtype=np.uint8)
+    data = np.zeros((4, 6, 8), dtype=np.uint8)
+
+    viewer = make_napari_viewer()
+
+    viewer.add_image(larger_data)
+    viewer.add_image(data)
+    viewer.dims.order = order
+
+    for actions in _LAYER_ACTIONS:
+        action = actions.get("napari:split_stack")
+        if action is not None:
+            break
+
+    split_action = action["action"]
```

Comment:
go ahead and just use `from napari.layers._layer_actions import _split_stack` and use it directly...